scenarios/hci.yaml

```diff
@@ -0,0 +1,69 @@
+---
+undercloud:
+  config:
+    - section: DEFAULT
+      option: undercloud_hostname
+      value: undercloud.localdomain
+    - section: DEFAULT
+      option: undercloud_timezone
+      value: UTC
+    - section: DEFAULT
+      option: undercloud_debug
+      value: true
+    - section: DEFAULT
+      option: container_cli
+      value: podman
+    - section: DEFAULT
+      option: undercloud_enable_selinux
+      value: false
+    - section: DEFAULT
+      option: generate_service_certificate
+      value: false
+  undercloud_parameters_override: "hci/hieradata_overrides_undercloud.yaml"
+  undercloud_parameters_defaults: "hci/undercloud_parameter_defaults.yaml"
+  ctlplane_vip: 192.168.122.99
+pre_oc_run:
+  - name: Deploy Ceph
+    type: playbook
+    source: "adoption_deploy_ceph.yml"
+    extra_vars:
+      vms_inventory: "{{ adoption_vms_inventory }}"
+cloud_domain: "localdomain"
+hostname_groups_map:
+  # map ansible groups in the inventory to role hostname format for
+  # 17.1 deployment
+  osp-computes: "overcloud-computehci"
+  osp-controllers: "overcloud-controller"
+roles_groups_map:
+  # map ansible groups to tripleo Role names
+  osp-computes: "ComputeHCI"
+  osp-controllers: "Controller"
+overcloud:
+  stackname: "overcloud"
+  args:
+    - "--override-ansible-cfg /home/zuul/ansible_config.cfg"
+    - "--templates /usr/share/openstack-tripleo-heat-templates"
+    - "--libvirt-type qemu"
+    - "--timeout 90"
+    - "--overcloud-ssh-user zuul"
+    - "--deployed-server"
+    - "--validation-warnings-fatal"
+    - "--disable-validations"
+    - "--heat-type pod"
+    - "--disable-protected-resource-types"
+  vars:
+    - "/home/zuul/deployed_ceph.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/docker-ha.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/podman.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/low-memory-usage.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/debug.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/manila-cephfsnative-config.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/enable-legacy-telemetry.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm.yaml"
+    - "/usr/share/openstack-tripleo-heat-templates/environments/cephadm/ceph-mds.yaml"
+  services_file: "hci/overcloud_services.yaml"
```

Comment:
not sure about adding a 'services_file' as an interface/variable here
the others, like vip_data network_data, ceph_osh_spec are 'expected' and different from 'normal' tripleo-heat-templates environment files.

but this services_file and also config_download (at least the way we are using it here like an environment file) are just environment files. I am not sure we want to keep them as top level interfaces.

there is no 'services file' notion in tripleo. that's not a reason to not have one here but do we need it, is it different from any other environment file.

Reply:
that's a good point, I haven't noticed that both this file and the config-download one are doing pretty much the same thing and could be merged into one, I've updated the patch merging them